pkg/reconciler/ingress/config/testdata/config-istio.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright 2018 The Knative Authors
```

Comment:
This should be a symlink

Reply:
I will make this a symlink in future PR, since the build test keeps failing if it's a symlink.